### DISCUSSION TITLE
Refresh Token

### DISCUSSION THREAD

**Discussion by HosMercury** [maxcountryman/axum-login#184] [Q&A] [answered]
I see you get the access token every time we log in.

what do you think about refresh _ token?

**Comment by maxcountryman** [comment, marked as answer]
It's possible to use refresh tokens by modifying the example implementation.

Please bear in mind the examples are only examples and by their very nature incomplete.

**Reply by JosiahParry** [reply]
@HosMercury did you figure out how to do this?

**Reply by HosMercury** [reply]
i do not use refresh token 
sorry to let you down
